codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/PresignerGenerator.kt

```diff
@@ -62,7 +61,7 @@ class PresignerGenerator : SwiftIntegration {
         val httpBindingResolver = protocolGenerator.getProtocolHttpBindingResolver(protocolGeneratorContext, protocolGenerator.defaultContentType)
 
         writer.openBlock("extension $inputType {", "}") {
-            writer.openBlock("public func presign(config: \$N, sigv4Config: \$D) -> \$T {", "}", AWSClientConfiguration, SigV4Config, SdkHttpRequest) {
+            writer.openBlock("public func presign(config: \$N, expiration: Int64) -> \$T {", "}", AWSClientConfiguration, SdkHttpRequest) {
```

Comment:
can you use a SwiftType here please instead of just directly stating the type? so should be:
```
writer.openBlock("public func presign(config: \$N, expiration: \$N) -> \$T {", "}", AWSClientConfiguration, SwiftTypes.Int64, SdkHttpRequest)
```

Reply:
Ahh yea, i was looking for SwiftTypes! that's what it was!